src/Main.elm

```diff
@@ -1370,8 +1401,13 @@ decodePresence =
             |> Json.Decode.andThen
                 (\type_ ->
                     case type_ of
-                        "System" ->
-                            Json.Decode.succeed (\name description -> System { name = name, description = description })
```

Comment:
If we can decode string `"system"` as `SystemRequired` then we’re backward compatible with existing json

Reply:
updated to handle `"System"` as `SystemRequired`

https://github.com/choonkeat/tiny-form-fields/pull/1/files#diff-2dd82f159d96fbfcd26fb7d885d25e0d54efde9e19a42494b416fa84a5aca568R1405